File: week 6/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if (h > 0 and h <= 8):
        break


for x in range(1, h+1):
    for y in range(h-x):
        print(" ", end="")
    for z in range(x):
        print("#", end="")
    print()

File: week 1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int height;
    do{
        height = get_int("How tall should the tower be? (1 - 8): ");
    }
    while(height < 1 || height > 8);

    for(int i=0; i<height; i++){
        for(int j=(height-1); j>i; j--){
            printf(" ");
        }
        for(int k=0; k<i; k++){
            printf("#");
        }
        printf("#  #");
        for(int l=0; l<i; l++){
            printf("#");
        }
        printf("\n");
    }
}
